verto/processors/VideoBlockProcessor.py

```diff
@@ -88,7 +88,7 @@ def extract_video_identifier(self, video_url):
         '''
 
         if re.match('.*?youtu\.{0,1}be(.com){0,1}', video_url) is not None:  # is a youtube url
-            video_url = re.sub(r'(.*?)(\?rel=0)', r'\g<1>', video_url)
+            video_url = re.sub(r'(.*?)(\?rel=0 )', r'\g<1>', video_url)
```

Comment:
I'm slightly confused on the change to this regex, could you explain it?

Reply:
um, I think that was an accident actually, don't know how I managed that, will fix now...